EXT_BeamPhysics.md

```diff
@@ -0,0 +1,264 @@
+Extension to the openPMD Standard for Describing Particle Beams and X-rays
+==========================================================================
+
+Version 2.0.0
+
+Overview
+--------
+
+The `BeamPhysics` extension to the openPMD standard is meant for describing particles and fields commonly encountered
+in accelerator physics simulations.
+
+How to Use this Extension
+-------------------------
+
+The `BeamPhysics` extension to the openPMD standard is indicated in a data file by the setting of the `openPMDextension` attribute:
+```
+  openPMDextension = BeamPhysics;SpeciesType
+```
+
+Note: The `speciesType` extension must be used when using the `BeamPhysics` extension.
```

Comment:
little typo: missing capital `S` in `speciesType` extension

Reply:
Fixed